content/3_Silver/Binary_Search.problems.json

```diff
@@ -77,6 +77,18 @@
         "usacoId": "419"
       }
     },
+    {
+      "uniqueId": "usaco-1159",
+      "name": "Connecting Two Barns",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=1159",
+      "source": "Silver",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["Binary Search", "DFS", "DSU", "2P"],
+      "solutionMetadata": {
```

Comment:
I'd prefer not to include this since:
 - binary search isn't necessary to solve this problem
 - graphs modules come after the binary search module